config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -260,7 +260,7 @@ periodics:
   tags:
   - "perfDashPrefix: kubemark-5000Nodes"
   - "perfDashJobType: performance"
-  interval: 12h
+  interval: 48h
```

Comment:
Not sure how to make each run on alternate days as we discussed, if you have any suggestions, please let me know.

Reply:
Use `cron` instead of `interval`. Here is an example - https://github.com/kubernetes/test-infra/blob/939c608dd2d9e6c974c2946a8c206e74f64a023a/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L3

Reply:
Done.